README.md

```diff
@@ -14,6 +14,11 @@ Either `apm install go-debug` or search for `go-debug` in the settings.
 
 If this fails you can still do it manually by using this guide: https://github.com/derekparker/delve/tree/master/Documentation/installation
 
+## Usage
+Use F5 to launch the debug window (the one on the right).
+At selected line press F9 to set (or toggle) break point.
+other [other function keys](#key-bindings).
```

Comment:
Just one more minor thing: you have repeated `other` here. Just remove the first one :)